src/components/molecules/Common/AssetModal/AssetCard/index.tsx

```diff
@@ -39,11 +39,7 @@ const AssetCard: React.FC<Props> = ({
       cardSize={cardSize}
       onClick={() => onCheck?.(!check)}>
       <ImgWrapper cardSize={cardSize}>
-        {url && /\.(jpg|jpeg|png|gif|svg|webp|GIF|JPG|JPEG|PNG|SVG|WEBP)$/.test(url) ? (
-          <PreviewImage url={url} />
-        ) : (
-          <Icon icon={icon} size={iconSize} />
-        )}
+        {!icon ? <PreviewImage url={url} /> : <Icon icon={icon} size={iconSize} />}
```

Comment:
I'm not sure, but what happens if someone tries to upload pdf file?
is there a place to handle this?

Reply:
PDF files are not supported as Assets. Many files never were supported but we had not restricted uploading enough, or rather correctly. I removed this Regex test because, if you see in the AssetContainer, what kind of icon is chosen (or no icon for "image") is based on a fileType check. If you check the `checkFileType` function it does the checking on the supported types.
For AssetCard, it'll be no icon for images, file icon for supported file types, and then if the file type doesn't pass either checks, it will show an unsupported file icon.